content/blog/math-in-javascript/index.md

```diff
@@ -0,0 +1,61 @@
+---
+{
+    title: "Math in JavaScript",
+    description: "Math in JavaScript may seem simple and sometimes easy, but sometimes what seems right is wrong.",
+    published: '',
+    authors: ['Jerico'],
+    tags: ['math', 'javascript'],
+    attached: [],
+    license: 'cc-by-4'
+}
+---
+
+# JavaScript: Mathematical Uses
```

Comment:
You don't need to have a header at the top of a post, otherwise you'll have double headers